ubuntu_image/testing/flake8.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (C) 2016 Barry Warsaw
+#
+# This project is licensed under the terms of the Apache 2.0 License.
```

Comment:
debian/copyright needs updating on this one.

Is there a way we could just depend on it later?